TheMovieDb/Modules/Home/HomeView.swift

```diff
@@ -7,10 +7,34 @@
 
 import UIKit
 
+enum FeedTypes: CaseIterable {
+    case trending
+    case nowPlaying
+    case popular
+    case topRated
+    case upcoming
+}
+
+private extension FeedTypes {
+    var feedTitle: String {
+        switch self {
+        case .trending: return "Trending"
+        case .nowPlaying: return "Now Playing"
+        case .popular: return "Popular"
+        case .topRated: return "Top Rated"
+        case .upcoming: return "Upcoming"
+        }
+    }
```

Comment:
Next session we'll discuss localizable files but for the moment it's OK

Reply:
Oks